sdk/formrecognizer/ai-form-recognizer/CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 
 ### Breaking Changes
 
-- Renamed the `getModel` and `listModels` methods of `DocumentModelAdministrationClient` to `getDocumentModel` and `listDocumentModels` respectively.
+- Renamed `DocumentModelAdministrationClient` methods to have the word `Document` in them, for example `getModel` and `listModels` are updated to `getDocumentModel` and `listDocumentModels` respectively.
```

Comment:
Just curious, does JS usually generalize things like there renames here? Or should each one be called out explicitly?

Reply:
Don't have a super strong preference, have done things both ways before.

Reply:
Ok sounds good then